pkg/config/log.go

```diff
@@ -55,25 +61,39 @@ func SetupLogger(logLevel, logFile, uri string, rfc, tls bool, pem string, logTo
 				`<custom name="syslog" formatid="syslog" data-uri="%s" data-tls="%v" />`,
 				uri,
 				tls,
-			)
+				)
 		} else {
 			syslogTemplate = `<custom name="syslog" formatid="syslog" />`
 		}
 		configTemplate += syslogTemplate
 	}
+
 	configTemplate += `</outputs>
-    <formats>
-        <format id="common" format="%%Date(%s) | %%LEVEL | (%%File:%%Line in %%FuncShort) | %%Msg%%n"/>`
+	<formats>`
+
+	if jsonFormat {
+		configTemplate += `<format id="std:json" format="{&quot;time&quot;:&quot;%%Date(%s)&quot;,&quot;level&quot;:&quot;%%LEVEL&quot;,&quot;msg&quot;:&quot;%%Msg&quot;}%%n"/>`
+	} else {
+		configTemplate += `<format id="common" format="%%Date(%s) | %%LEVEL | (%%File:%%Line in %%FuncShort) | %%Msg%%n"/>`
+	}
 	if syslog {
 		if rfc {
-			configTemplate += `<format id="syslog" format="%%CustomSyslogHeader(20,true) %%LEVEL | (%%RelFile:%%Line) | %%Msg%%n" />`
+			if jsonFormat {
```

Comment:
Since we can define multiple format, I would remove the conditions and define them all with different id and activate the right one the the formats list above.